lib/shelley/exe/shelley-test-cluster.hs

```diff
@@ -236,6 +236,8 @@ main = do
     whenReady tr trCluster db (RunningNode socketPath block0 (gp, vData)) = do
         let tracers = setupTracers (tracerSeverities (Just Info)) tr
         listen <- walletListenFromEnv
+        prometheusUrl <- (maybe "none" (\(h, p) -> T.pack h <> ":" <> toText @(Port "Prometheus") p)) <$> getPrometheusURL
+        ekgUrl <- (maybe "none" (\(h, p) -> T.pack h <> ":" <> toText @(Port "EKG") p)) <$> getEKGURL
```

Comment:
These lines are longer than our style guide. In-particular, the important `get*URL` are way outside 80 chars.